src/graphql/schema/createGqlSchema.ts

```diff
@@ -40,6 +43,6 @@ export default function createGqlSchema (inventory: Inventory, options: SchemaOp
 
   return new GraphQLSchema({
     query: getQueryGqlType(buildToken),
-    mutation: getMutationGqlType(buildToken),
+    mutation: getMutationGqlType(buildToken, options.disableDefaultMutations),
```

Comment:
You’d also want to add a line to the [`BuildToken`](https://github.com/calebmer/postgraphql/blob/c857057f9f9bddb5af398ef893dd20bcfad4f408/src/graphql/schema/BuildToken.ts#L22-L30) type, where you could make the option required.